docs/user-guide/tls-configuration.md

```diff
@@ -0,0 +1,85 @@
+# Customizing Native TLS
+
+Zowe's servers have built-in TLS support to enable HTTPS connections.
+This is the default, and an alternative to using AT-TLS which is [documented here](./at-tls-configuration).
+
+:::info Required roles: security administrator
+:::
+
+## Server Parameters
+
+Each Zowe server can be customized either by defining attributes within the `zowe.network.server` object of the Zowe YAML configuration file. The same object can be put within an individual component's configuration, such as `components.zss.zowe.network.server` for ZSS, which will allow you to customize each component separate from others.
+Extensions are recommended to adhere to this configuration, but you must check with documentation for extensions to be sure.
```

Comment:
I think you need a blank line here to create a line break.

Reply:
Done thanks!